crates/modelardb_server/src/storage/mod.rs

```diff
@@ -228,13 +228,22 @@ impl StorageEngine {
             )?;
         }
 
-        Ok(Self {
+        let mut storage_engine = Self {
             uncompressed_data_manager,
             compressed_data_manager,
             memory_pool,
             join_handles,
             channels,
-        })
+        };
+
+        // Start the task that transfers data periodically if a data transfer component exists and
+        // time-based data transfer is enabled. Errors are ignored since an error is only returned
+        // if a data transfer component does not exist.
```

Comment:
Maybe change "if a data transfer component does not exist." to something like "if the data transfer component has not been enabled." as it always exists.

Reply:
Fixed.